napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -1295,19 +1313,29 @@ def _update_plugin_count(self):
 
         available_count = len(self._plugin_data) - self.installed_list.count()
         available_count = available_count if available_count >= 0 else 0
-        available_count_visible = self.available_list.count_visible()
-        if available_count == available_count_visible:
+
+        if self._plugins_found == 0:
+            self.avail_label.setText(
+                trans._(
+                    "{amount} plugins available in the napari Hub",
```

Comment:
```suggestion
                    "{amount} plugins available on the napari hub",
```
on napari-hub.org it's stylized as `napari hub`